ozaria/engine/cinematic/dialogsystem/DialogSystem.js

```diff
@@ -104,18 +104,30 @@ class SpeechBubble {
     this.id = `speech-${_id++}`
     const parser = new DOMParser()
     const html = parser.parseFromString(htmlString, 'text/html')
+
     const textDiv = html.body.firstChild
-    textDiv.style.display = 'inline-block'
-    textDiv.style.position = 'absolute'
-    textDiv.style.maxWidth = SPEECH_BUBBLE_MAX_WIDTH
-    textDiv.style.opacity = 0
-    textDiv.id = this.id
-    textDiv.className = `cinematic-speech-bubble-${side}`
+    const speechBubbleDiv = document.createElement('div')
```

Comment:
We now have a speech bubble component!  Maybe something we could use here?  Not necessarily for this PR

Reply:
Will need a few tweaks for left and right speech bubbles.

Reply:
That is definitely an improvement that could be added. These are very manual, and attaching vue components would be much nicer. Moving this to be more declerative is something worth pursuing.

Does it also support the 9-slice? Would you be able to link me to it?
Thank you!

Reply:
It's done with pure CSS right now but could be changed!  https://github.com/codecombat/codecombat/blob/master/ozaria/site/components/common/BaseSpeechBubble.vue